docs/source/release_notes/release-notes.rst

```diff
@@ -42,8 +42,11 @@ develop
     *    - Type
          - Change
 
-    *    -
-         -
+    *    - |improved| |devbreak|
+         - Improved performance by preventing excessive reads from the _namespace table when initializing SweepStrategyManager.
```

Comment:
Sorry for the delay. I am not sure how this will prevent DB reads from the namespace table. Can we add tests that ensure that we have less DB round trips.